### PR TITLE
MM-18903 - Removal of unused status flags in Integrations requests state object

### PR DESCRIPTION
**Pull request by nadalfederer**
Remove unused status flags from https://github.com/mattermost/mattermost-redux/blob/master/src/reducers/requests/integrations.ts

 Fixes https://github.com/mattermost/mattermost-server/issues/12519

### CODE REVIEW

**Comment by nadalfederer** [comment]
It's ready.

File: src/actions/integrations.ts [hunk 1]
```diff
@@ -53,17 +47,11 @@ export function getIncomingHooks(teamId = '', page = 0, perPage: number = Genera
...
-        dispatch({type: IntegrationTypes.DELETE_INCOMING_HOOK_REQUEST, data: {}}, getState);
-
         try {
             await Client4.removeIncomingWebhook(hookId);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
 
-            dispatch(batchActions([
-                {type: IntegrationTypes.DELETE_INCOMING_HOOK_FAILURE, error},
-                logError(error),
```
Comment (sudheerDev) [inline src/actions/integrations.ts]: @nadalfederer Hey, We still like to dispatch the log errors here. Would you mind adding them back?

**Comment by sudheerDev** [comment]
> It's ready. Is this the right approach?

Yup, It was perfect 🎉

File: src/actions/integrations.ts (hunk 1 above)
Comment (nadalfederer) [reply src/actions/integrations.ts]: Thanks for the review. I fixed it

File: src/actions/integrations.ts [hunk 2]
```diff
@@ -52,6 +52,10 @@ export function removeIncomingHook(hookId: string): ActionFunc {
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
 
+            dispatch(batchActions([
+                {type: IntegrationTypes.DELETE_INCOMING_HOOK_FAILURE, error},
```
Comment (sudheerDev) [inline src/actions/integrations.ts]: Sorry we dont need the failure action here. 

i meant adding back just the log error
`dispatch(logError(error));`

Comment (nadalfederer) [reply src/actions/integrations.ts]: thanks, I fixed it

File: src/actions/integrations.ts [hunk 3]
```diff
@@ -387,43 +315,31 @@ export function getAuthorizedOAuthApps(): ActionFunc {
...
-
         return {data};
     };
 }
 
 export function deauthorizeOAuthApp(clientId: string): ActionFunc {
     return bindClientFunc({
         clientFunc: Client4.deauthorizeOAuthApp,
-        onRequest: IntegrationTypes.DEAUTHORIZE_OAUTH_APP_REQUEST,
         onSuccess: IntegrationTypes.DEAUTHORIZE_OAUTH_APP_SUCCESS,
```
Comment (jespino) [inline src/actions/integrations.ts]: You don't need the `onSuccess` here, the `DEAUTHORIZE_OAUTH_APP_SUCCESS` is not used anywhere (I think).

File: src/actions/integrations.ts [hunk 4]
```diff
@@ -251,9 +207,7 @@ export function editCommand(command: Command): ActionFunc {
 export function executeCommand(command: Command, args: Array<string>): ActionFunc {
     return bindClientFunc({
         clientFunc: Client4.executeCommand,
-        onRequest: IntegrationTypes.EXECUTE_COMMAND_REQUEST,
         onSuccess: IntegrationTypes.EXECUTE_COMMAND_SUCCESS,
```
Comment (jespino) [inline src/actions/integrations.ts]: you don't need the onSuccess here

File: src/action_types/integrations.ts [hunk 5]
```diff
@@ -3,97 +3,10 @@
...
-    DELETE_OAUTH_APP_REQUEST: null,
-    DELETE_OAUTH_APP_SUCCESS: null,
-    DELETE_OAUTH_APP_FAILURE: null,
-
-    UPDATE_OAUTH_APP_REQUEST: null,
-    UPDATE_OAUTH_APP_SUCCESS: null,
-    UPDATE_OAUTH_APP_FAILURE: null,
-
-    DEAUTHORIZE_OAUTH_APP_REQUEST: null,
     DEAUTHORIZE_OAUTH_APP_SUCCESS: null,
```
Comment (jespino) [inline src/action_types/integrations.ts]: you don't need this here.

File: src/action_types/integrations.ts [hunk 6]
```diff
@@ -3,97 +3,10 @@
...
-    DEAUTHORIZE_OAUTH_APP_FAILURE: null,
 
-    EXECUTE_COMMAND_REQUEST: null,
     EXECUTE_COMMAND_SUCCESS: null,
```
Comment (jespino) [inline src/action_types/integrations.ts]: you don't need this here.

**Comment by sudheerDev** [comment]
@jespino Requesting a review again from you as you had comments after your approval.

**Comment by enahum** [comment]
🎉 this PR broke the mobile app

**Pull request merged**